### DISCUSSION TITLE
Anybody willing to try a fight against my bot ?

### DISCUSSION THREAD

**Discussion by AengelStudio** [SebLague/Chess-Challenge#155] [Ideas]
For testing purposes (my bot is a bit rudimentary for the moment) , this is my discord if you are interested : diamoundz

**Comment by BoboNovi** [comment]
I sent you a friend request my discord is @notbobo_

**Comment by SebastianBGg** [comment]
Would be fun to try out, added you discord is star

**Comment by DaveAxiom** [comment]
@Diamoundz Could you share your creation as a Github [gist](https://gist.github.com/)? That Github feature seems to be best suited for single source code file sharing.
